services/provider/server/server.go

```diff
@@ -485,9 +485,9 @@ func validateTicket(ticket string, pubKey *rsa.PublicKey) error {
 	return nil
 }
 
-// FulfillStorageClassClaim RPC call to create the StorageclassClaim CR on
+// StorageClassRequest RPC call to create the StorageClassRequest CR on
 // provider cluster.
-func (s *OCSProviderServer) FulfillStorageClassClaim(ctx context.Context, req *pb.FulfillStorageClassClaimRequest) (*pb.FulfillStorageClassClaimResponse, error) {
+func (s *OCSProviderServer) StorageClassRequest(ctx context.Context, req *pb.StorageClassRequestRequest) (*pb.StorageClassRequestResponse, error) {
```

Comment:
the RPC calls should remain the same only the CRD create/Delete/Get API calls need to be changed

Reply:
got it.

Reply:
Is this supposed to change terminology? We are still fulfilling a storageclassclaim (not request) so I am not sure why is this change here.